run-qa-check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I'd keep consistency with the other modules and call it `run-qa-checks` (standardization will also help us with the pre-commit hooks we discussed recently).

Reply:
Done! :-)